spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/JdbcChannelMessageStore.java

```diff
@@ -384,8 +384,8 @@ public void afterPropertiesSet() {
 			this.messageRowMapper = new MessageRowMapper(this.deserializer, this.lobHandler);
 		}
 
-		if (this.jdbcTemplate.getFetchSize() != 1 && LOGGER.isWarnEnabled()) {
-			LOGGER.warn("The jdbcTemplate's fetch size is not 1. This may cause FIFO issues with Oracle databases.");
+		if (this.jdbcTemplate.getFetchSize() != 1) {
+			LOGGER.warn(() -> "The jdbcTemplate's fetch size is not 1. This may cause FIFO issues with Oracle databases.");
```

Comment:
Don't need `Supplier` wrapper over here - just plain string constant.

Would you mind to revise all other log messages in this class in favor of discovered that `LogMessage.format()`?

Reply:
Ok

Reply:
There is no reason to comment on the review if you agree with the proposal.
Otherwise I get email for every single your `Ok` 😄